### PR TITLE
Split XCTest into a new product

### PR DESCRIPTION
**Pull request by jarnal**
### Context

This PR splits `SwiftyMocky` in two different products:
- `SwiftyMocky`
- `SwiftyMockyXCTest`

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -18,8 +18,8 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   Sourcery: 47b30d2494896127ff264ecb19efb3643521fcb9
-  SwiftyMocky: f3ac32a103b6fd945e08cb2d3a88a03185825737
+  SwiftyMocky: f327448da3a46db509fe09451e5cf01995d8f94d
 
 PODFILE CHECKSUM: 024f8b7ce6c9eed2b70ca54b9fcecda8c6cb6381
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.12.0
```
Comment (ios-qonto) [inline Podfile.lock]: Oleksii: why do we decrease the pods version?

Comment (jarnal) [reply Podfile.lock]: You are right, will revert this change.

**Review by ios-qonto — APPROVED** [review]
Approved by Oleksii Bezrodnyi ✅

**Review by MariuszWisniewski — APPROVED** [review]

**Comment by MariuszWisniewski** [comment]
LGTM ✅

**Pull request merged**